src/DigitalProductPassport/LogisticsEmissions_v0.1.py

```diff
@@ -0,0 +1,252 @@
+from enum import Enum
+from typing import List, Optional
+
+from definition_tooling.converter import CamelCaseModel, DataProductDefinition
+from pydantic import Field
+
+
+class FreightType(str, Enum):
+    PALLETIZED = "Palletized"
+    DRY_BULK = "Dry bulk"
+    LIQUID_BULK = "Liquid bulk"
+    CONTAINERIZED = "Containerized"
+    VEHICLE_TRANSPORT = "Vehicle transport"
+    HEAVY_CARGO = "Heavy cargo"
+    LIGHT_CARGO = "Light cargo"
+
+
+class RoadLegFreightCondition(str, Enum):
+    AMBIENT = "Ambient"
+    TEMPERATURE_CONTROLLED = "Temperature-controlled"
+
+
+class SeaLegFreightCondition(str, Enum):
+    AMBIENT = "Ambient"
+    TEMPERATURE_CONTROLLED = "Temperature-controlled"
+    MIXED = "Mixed ambient and temperature-controlled"
+
+
+class JourneyType(str, Enum):
+    LONG_HAUL = "Long-haul"
+    COLLECTION_AND_DELIVERY = "Collection and delivery"
+
+
+class ContractType(str, Enum):
+    SHARED_TRANSPORT = "Shared transport"
+    DEDICATED_CONTRACT = "Dedicated contract"
+
+
+class ServiceType(str, Enum):
+    SCHEDULED = "Scheduled"
+    TRAMP = "Tramp"
+
+
+class VesselType(str, Enum):
+    BULK_CARRIER = "Bulk carrier"
+    CHEMICAL_TANKER = "Chemical tanker"
+    GENERAL_CARGO = "General cargo"
+    RO_RO = "Ro-Ro"
+    LIQUEFIED_GAS_TANKER = "Liquefied gas tanker"
+    OIL_TANKER = "Oil tanker"
+    OTHER_LIQUEFIED_TANKER = "Other liquefied tanker"
+    CONTAINER = "Container"
+    VEHICLE_CARRIER = "Vehicle carrier"
+
+
+class EmissionsPerTCE(CamelCaseModel):
+    description: Optional[str] = Field(
+        None,
+        max_length=250,
+        title="Description",
+        description="The description of the transport chain element (TCE) related to the leg",
+        example="Loading for road transport",
+    )
+    emissions: Optional[float] = Field(
+        None,
+        title="Emissions",
+        description="The green house gas (GHG) emissions of the transport "
+        "chain element related to the road tranport leg in CO2e tonnes",
```

Comment:
The spreadsheet has different texts for the descriptions for SeaLeg and RoadLeg. We'll need to check if we can unify them in some way so the same model could be used in both cases.